ledger-api/grpc-definitions/com/daml/ledger/api/v1/admin/party_management_service.proto

```diff
@@ -113,13 +118,35 @@ message AllocatePartyRequest {
   // have to be unique.
   // Optional
   string display_name = 2;
```

Comment:
deprecate it, use metadata instead
or
just comment: consider using metadata instead